teams.md/docs/main/teams/configuration/agents-toolkit.md

```diff
@@ -1,11 +1,11 @@
 ---
-sidebar_position: 3
+sidebar_position: 1
 summary: Automate Teams app development with Microsoft 365 Agents Toolkit for manifest management, sideloading, and deployment.
 ---
 
 # Microsoft 365 Agents Toolkit
 
-Microsoft 365 Agents Toolkit provides a powerful VS Code extension and CLI tool that helps automate important tasks like manifest management, sideloading, deployment, and provisioning. If you encounter any issues while using it (such as problems with the extension, running apps, deployment and provisioning, or debugging via F5), please report them in the [Agents Toolkit GitHub repository](https://github.com/OfficeDev/microsoft-365-agents-toolkit).
+Microsoft 365 Agents Toolkit provides a powerful VS Code extension and CLI tool that helps automate important tasks like provisioning and deployment. The project is maintained separately in this repository - [Agents Toolkit GitHub repository](https://github.com/OfficeDev/microsoft-365-agents-toolkit).
```

Comment:
better, less confusing wording.

Reply:
ATK current seems to still build multi-tenant app. Sent a message to them to ask.

Reply:
currently they add code to handle that.... once this goes in, they'll clear that up so it uses env variables